src/applications/facility-locator/components/search-results/LocationPhoneLink.jsx

```diff
@@ -36,6 +34,25 @@ const renderPhoneNumber = (
         </a>
       ) : null}
     </div>
+  ) : (
+    <dfn>
+      {from === 'FacilityDetail' && <i className={`fa fa-${icon}`} />}
```

Comment:
I think this component could be cleaner too if it just checked a `isFacilityDetail` or `showDetails` argument instead doing the comparison with the floating string.